routes/resources.js

```diff
@@ -86,7 +86,10 @@ module.exports = function (app) {
   })
 
   app.get(`/api/v${VER}/discovery/resources/:uri\.:ext?`, function (req, res) {
+    var gatheredParams = gatherParams(req, ['uri', 'items_size', 'items_from'])
     var params = { uri: req.params.uri }
+    if (gatheredParams.items_size) params.items_size = gatheredParams.items_size
```

Comment:
Edge case, but if someone really wants to set `items_size` to `0`, that'll be falsey and so it won't make it into the query. Should the check be `if (Number.isInteger(gatheredParams.items_size))`?

Reply:
I think the params are actually strings, I've changed this to `Number.isInteger(parseInt(...))`